src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { urlPartReplacer } from "Shared/urlParams"
+
+import { UsePlatforms } from "../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../types"
+import { conditions } from "../utils/conditions"
+
+const waveHeight = new Set(conditions.waveHeight)
+const windSpeed = new Set(conditions.windSpeed)
+
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+export const Superlatives: React.FunctionComponent = () => {
+  const unitSystem = useUnitSystem()
+
+  const lastHour = anHourAgoRounded()
+
+  return (
+    <UsePlatforms>
+      {({ platforms }) => <ShowSuperlatives laterThan={lastHour} {...{ platforms, unitSystem }} />}
+    </UsePlatforms>
+  )
+}
+
+interface ShowSuperlativesProps {
+  platforms: PlatformFeature[]
+  unitSystem: UnitSystem
+  laterThan: Date
+}
+
+/**
+ * Show the highest winds and biggest waves for selected platforms
+ *
+ * Only shows if both wind and wave data is available
+ *
+ * @param platforms A selection of PlatformFeatures
+ * @param unitSystem unit system to display with
+ * @param laterThan a date to make sure all the readings are more recent then
+ */
+export const ShowSuperlatives: React.FunctionComponent<ShowSuperlativesProps> = ({
+  platforms,
+  unitSystem,
+  laterThan,
+}) => {
+  const { platform: windPlatform, timeSeries: windTimeSeries } = findHighestCondition(platforms, laterThan, windSpeed)
+  const { platform: wavePlatform, timeSeries: waveTimeSeries } = findHighestCondition(platforms, laterThan, waveHeight)
+
+  if (windPlatform && windTimeSeries && wavePlatform && waveTimeSeries) {
+    return (
+      <Card style={{ marginTop: "1rem" }}>
+        <CardHeader>
+          <h5>Latest Conditions</h5>
+        </CardHeader>
+
+        <CardBody>
+          <Row>
+            <Col>
+              <HighestConditions
+                title="Highest Winds"
+                platform={windPlatform}
+                timeSeries={windTimeSeries}
+                unitSystem={unitSystem}
+              />
+            </Col>
+
+            <Col>
+              <HighestConditions
+                title="Biggest Waves"
+                platform={wavePlatform}
+                timeSeries={waveTimeSeries}
+                unitSystem={unitSystem}
+              />
+            </Col>
+          </Row>
+        </CardBody>
+      </Card>
+    )
+  }
+
+  return null
+}
+
+/**
+ * Filter down and find the most extreme values from the selected set of standard names
+ *
+ * @param platforms Platforms to search through
+ * @param laterThan Time window to make sure found timeSeries are before
+ * @param compareSet Set of standard names to include in search
+ */
+function findHighestCondition(
+  platforms: PlatformFeature[],
+  laterThan: Date,
+  compareSet: Set<string>
+): { platform?: PlatformFeature; timeSeries?: PlatformTimeSeries } {
+  let highestPlatform: PlatformFeature | undefined = undefined
+  let highestTimeSeries: PlatformTimeSeries | undefined = undefined
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)